test/retry.js

```diff
@@ -4,6 +4,8 @@ import ky from '..';
 
 const fixture = 'fixture';
 const defaultRetryCount = 2;
+const retryAfterOn413 = 2;
+const lastTried413access = Date.now();
```

Comment:
This one is only used in one test, so should be defined inside that test.

Reply:
@sindresorhus `retryAfterOn413` and `lastTried413access` used in two tests
https://github.com/sindresorhus/ky/blob/cec5ab882a5ce1c60edb8ed504de48c77c8b09d5/test/retry.js#L86-L107

https://github.com/sindresorhus/ky/blob/cec5ab882a5ce1c60edb8ed504de48c77c8b09d5/test/retry.js#L109-L131